### DISCUSSION TITLE
GraphQL subscriptions support / WS support

### DISCUSSION THREAD

**Discussion by josias-r** [directus/directus#4127] [Feature Requests]
From what I can see there is no support for GraphQL subscriptions or the WebSocket protocol in general. I would absolutely love to see this feature.

I guess GraphQL mutations are the next step, but I would love this feature to be considered afterwards 😁

Thanks for this awesome project nonetheless ❤️

**Comment by rijkvanzanten** [comment]
Please let me know your thoughts regarding a generic WebSocket implementation here: https://github.com/directus/directus/discussions/3221 😄

**Comment by rijkvanzanten** [comment]
Closing in favor of https://github.com/directus/directus/discussions/3255. (This is coming very soon! 😄 )
